common/test/src/View/SnapshotTest.cpp

```diff
@@ -49,17 +50,39 @@ namespace TrenchBroom {
             ASSERT_NE(nullptr, texture);
             ASSERT_EQ(6u, texture->usageCount());
 
-            for (const Model::BrushFace& face : brushNode->brush().faces())
+            for (const Model::BrushFace& face : brushNode->brush().faces()) {
                 ASSERT_EQ(texture, face.texture());
-
-            document->translateObjects(vm::vec3(1, 1, 1));
-            ASSERT_EQ(6u, texture->usageCount());
-
-            document->undoCommand();
-            ASSERT_EQ(6u, texture->usageCount());
-
-            for (const Model::BrushFace& face : brushNode->brush().faces())
+            }
+            
+            SECTION("translate brush") {
+                document->translateObjects(vm::vec3(1, 1, 1));
+                ASSERT_EQ(6u, texture->usageCount());
+
+                document->undoCommand();
+                ASSERT_EQ(6u, texture->usageCount());
+            }
+            SECTION("select top face, move texture") {
```

Comment:
Missing newline?

Reply:
you want spaces between SECTION's?

Reply:
I usually put them, yeah, because it reflects how separate they are. What do you think?

Reply:
Sure, that makes sense. I was just thinking of it like
```
if (..) {

}
if (..) {

}
```